kernel/content/versionview.php

```diff
@@ -13,6 +13,8 @@
 $LanguageCode = htmlspecialchars( $LanguageCode );
 $viewParameters = array( 'offset' => $Offset );
 
+ezpEvent::getInstance()->notify('content/cache/translations', array($ObjectID, $EditVersion, [$LanguageCode]));
```

Comment:
Afaik this is the wrong place to clear cache, and also even name implies it's a cache event, however what user is doing here is not initiate clear cache, but rather viewing a preview.

A bit of a guess, but I would think there needs to be a event (if there isn't already) fired when system stores draft, which is afaik done before redirecting to previewing it. right?

Reply:
@andrerom Thanks for your feedback. I have moved the code to `attribute_edit.php`.